lib/state.js

```diff
@@ -0,0 +1,66 @@
+const debug = require('debug')('snyk:nodejs:state');
```

Comment:
In general - the name `state.js` is not indicative, even confusing. What about `runtimeContext`? Or whatever you like

Reply:
the entire agent is "runtime" - so we could append "runtime" to every module here. "runtimeDebuggerWrapper", "runtimeTransmitter" etc. therefore it's kinda redundant.
we could try `context` if you like, but I like `state` better.
this is literally the `state` of the agent, so I think it's as indicative as it gets